WordPress/openverse
# Convert the last components to script-setup

## Fixes



Fixes #4678 by @obulat

## Description


This PR is the last one, converting all the rest of the components.

In addition to converting items to script setup (using `defineProps`, dropping of `defineComponent`, removing returns), it removes the sizes from `VInputField` because we no longer use them.

## Testing Instructions

Check that the code changes make sense and the CI passes.
Check that the content page; the single result pages; the footer work correctly and have no warnings.
Specifically, check:
- the sources table on `/sources` and it's sorting icon.
- the collection pages' header
- content pages (about, sources, etc)
- footer on all layouts (about, search, single result, error page)
- search bar in the header
- icon buttons
- radio inputs in the report form
- skip to content button
- error page (http://localhost:8443/any-gibberish-will-show-you-the-error-page)
- global audio player

## Checklist


      PRs) or the media properties generator (`ov just catalog/generate-docs media-props`
      for the catalog or `ov just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
